extensions/amp-form/0.1/amp-form.js

```diff
@@ -508,6 +563,39 @@ export class AmpForm {
     return this.form_.querySelectorAll('[type="hidden"][data-amp-replace]');
   }
 
+  /**
+   * Handle successful presubmit tasks
+   * @param {!ActionTrust} trust
+   * @return {!Promise}
+   */
+  handlePresubmitSuccess_(trust) {
+    if (this.xhrAction_) {
+      return this.handleXhrSubmit_(trust);
+    } else if (this.method_ == 'POST') {
+      this.handleNonXhrPost_();
+    } else if (this.method_ == 'GET') {
+      this.handleNonXhrGet_(event);
```

Comment:
Is this the `window.event` variable? I don't see it defined. Using the global `event` should be avoided

https://developer.mozilla.org/en-US/docs/Web/API/Window/event

Reply:
Ah! You are complete correct! Must have forgot to pass that down when I moved this into its own function 😮 Thanks! Will fix that now! Surprised check types didn't catch this